### ISSUE TITLE
Rydde, slå saman, endre?

### ISSUE THREAD

**Issue by Hypsibius** [realfagstermer/realfagstermer#1288]
![image](https://user-images.githubusercontent.com/9350370/40611211-ba06385e-6275-11e8-9cb8-210641afbda5.png)

Og ingen av dei dekte heilt det eg var på jakt etter *sukk*

**Comment by danmichaelo** [comment]
Hva var du ute etter, da?

**Comment by danmichaelo** [comment]
Foreløpig: Slo sammen `Datastøttet læring BF Databasert læring` (det første er sikkert litt hippere) og la til `BF Digital læring`.

@Hypsibius, hva var det du var på jakt etter?

**Comment by Hypsibius** [comment]
Ja, sei det? Det kan eg ikkje lenger hugse. Må hugse å skrive det ned til neste gong
